frontend/packages/console-shared/src/types/index.ts

```diff
@@ -0,0 +1,5 @@
+export type LoadingList<T> = {
```

Comment:
We already have this defined here:

https://github.com/openshift/console/blob/master/frontend/public/components/utils/index.tsx#L94-L98

Reply:
Thanks, missed that..